LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Muhammad Ahmad Saeed
```

Comment:
Hi @ehmaddd ,
Please include the name of other authors in the license file.

Reply:
The broken link is fixed

Reply:
Included as instructed